resources/views/list-view/filters.blade.php

```diff
@@ -0,0 +1,26 @@
+{{-- list-view.filters.blade
+
+Renders the search input and the filters dropdown
+You can customize all the html and css classes but YOU MUST KEEP THE BLADE AND LIVEWIERE DIRECTIVES
+UI components used:
+  - form/input-group
+  - dropdown --}}
+
+<div class="flex flex-wrap">
+  {{-- Search input --}}
+  <div class="flex-grow {{ isset($sortableBy) && $sortableBy->isNotEmpty() ? 'w-full lg:w-auto' : '' }}">
+    @include('laravel-views::components.filters.search')
+  </div>
+
+  {{-- Sorting --}}
+  @if (isset($sortableBy) && $sortableBy->isNotEmpty())
```

Comment:
Not sure if I like how it looks on 'md' or below, what if we pull the sorting button (regular size) to the left and the filters button to the right?